doc/glossary.rst

```diff
@@ -949,6 +962,10 @@ Class APIs and Estimator Types
         A purely :term:`transductive` transformer, such as
         :class:`manifold.TSNE`, may not implement ``transform``.
 
+    resampler
+    resamplers
+        An estimator supporting :term:`fit_resample`.
```

Comment:
```suggestion
        An estimator supporting :term:`fit_resample`. This can be used in a
        :class:`ResampledTrainer` to resample, augment or reduce the training
        dataset passed to another estimator.
```